homeassistant/components/isy994/entity.py

```diff
@@ -94,3 +117,28 @@ def __init__(self, name: str, status, actions=None) -> None:
         super().__init__(status)
         self._name = name
         self._actions = actions
+
+    @property
+    def device_state_attributes(self) -> Dict:
+        """Get the state attributes for the device."""
+        attr = {}
+        if self._actions:
+            attr["actions_enabled"] = self._actions.enabled
+            if attr["actions_last_finished"] != EMPTY_TIME:
+                attr["actions_last_finished"] = self._actions.last_finished
+            if attr["actions_last_run"] != EMPTY_TIME:
```

Comment:
Thank you for fixing the empty attributes issue

Reply:
Updated in last commit to check the right thing.